modules/guides/pages/streams_mode/using_config_files.adoc

```diff
@@ -13,7 +13,7 @@ A stream configuration should only include the base stream component fields (`in
 
 [,bash,subs="attributes+"]
 ----
-rpk connect run -r "./resources/prod/*.yaml" streams ./stream_configs/*.yaml
+rpk connect streams -r "./resources/prod/*.yaml" ./stream_configs/*.yaml
```

Comment:
Is this correct? See line 60. I tried different variants of this command to complete the walkthrough but got errors in each case.